.devcontainer/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM mcr.microsoft.com/vscode/devcontainers/java:11
+FROM mcr.microsoft.com/vscode/devcontainers/java:21
```

Comment:
The presto-kerberos-integration-test.yml GitHub workflow depends on this container and that test was failing. This matches the versions found in the Dockerfile in the root folder (Java 21, Node 22, Clojure 1.12).